### PR TITLE
Fixes #288: Include Material Module

### LINKED ISSUE #288: Inclulde Angular Material and theming in the project
* Add @angular/material2 to the project and set up theming

### PR DESCRIPTION
**Pull request by hemantjadon**
* Angular Material2 include

* Set Theming

* Some minor indentation changes

* CLI version bumped to RC2 from RC0

**Closes #288**

### CODE REVIEW

**Comment by Achint08** [comment]
@hemantjadon  Add about angular2-material in the technology stack so that beginners know about it. :)

File: src/app/feed/feed-header/feed-header.component.scss [hunk 1]
```diff
@@ -21,7 +21,7 @@
 			font-family: "Courier New";
 			padding:15px 15px;
 			font-size:35px;
-			color: #C8254D;
+			color: #e91e63;
```
Comment (uttu357) [inline src/app/feed/feed-header/feed-header.component.scss]: Why is the color changed?

Comment (hemantjadon) [inline src/app/feed/feed-header/feed-header.component.scss]: In accordance with the indigopink theme from angular material. We can add custom theme later. Its just a different shade of pink.

**Comment by hemantjadon** [comment]
@rishiraj824 Its just indentation correction.

File: src/app/feed/feed-header/feed-header.component.scss (hunk 1 above)
Comment (uttu357) [inline src/app/feed/feed-header/feed-header.component.scss]: Ohkz.. :+1:

**Comment by uttu357** [comment]
LGTM Should be merged after resolving the merge conflicts.. :)

**Comment by uttu357** [comment]
@hemantjadon Please check that the commit message has been altered during the rebase! :)

**Comment by hemantjadon** [comment]
Please followup on #300 Rebased wron

**Pull request closed without merging**